doc/classes/HeightMapShape3D.xml

```diff
@@ -9,9 +9,23 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="get_max_height" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the largest height value found in [member map_data].
+			</description>
+		</method>
+		<method name="get_min_height" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the smallest height value found in [member map_data].
```

Comment:
Might be a good idea to explicitly mention that these return cached value / they don't do any calculation.

Reply:
But can you change the map_data to get things out-of-sync? I dont think so since it uses a packed array users need to set the array again to update anything.

Reply:
I wasn't suggesting they can be out of sync or anything like that (no idea if they actually can). I only meant that from the user POV I have no idea whether min/max heights are precalculated or whether they're calculated on the fly when I'm calling these methods. It would be nice to know that they are cheap to call (`O(1)`, not `O(map_data.size())`).
So an additional note would make sense to me, something like:
```suggestion
				Returns the largest height value found in [member map_data] (the value is precalculated, calling this method does not involve iterating over [member map_data]).
			</description>
		</method>
		<method name="get_min_height" qualifiers="const">
			<return type="float" />
			<description>
				Returns the smallest height value found in [member map_data] (the value is precalculated, calling this method does not involve iterating over [member map_data]).
```